_data/changelogs/template-documentation.yml

```diff
@@ -0,0 +1,16 @@
+title: Documentation page
+type: template
+items:
+  - date: NNNN-NN-NN
+    summary: Visual order of the Documentation page matches the HTML order.
+    summaryAdditional:  Now, there are two separate side navigation components for mobile and desktop that are displayed or hidden depending on viewport widths.
```

Comment:
## Optional
I don't think this statement is necessary, but I also don't think it causes harm. I lean towards removing it for the sake of brevity, but it could go either way. 

```suggestion
    summaryAdditional:  
```